package/base-files/files/lib/functions/system.sh

```diff
@@ -72,6 +72,24 @@ get_mac_ascii() {
 	[ -n "$mac_dirty" ] && macaddr_canonicalize "$mac_dirty"
 }
 
+mmc_get_mac_ascii() {
```

Comment:
This function already exists in this file in line 200, see https://github.com/openwrt/openwrt/pull/14350/files#diff-9d8640936a163ac354f846da8410231e303d9d63bb3cfb7f83685cd49b111cc3L200